wiki/Beatmapping/Approach_Rate/en.md

```diff
@@ -0,0 +1,49 @@
+---
+tags:
+  - AR
+  - reading
+---
+
+# Approach rate
+
+*For suggested AR values, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
+
+**Approach rate** (***AR***) exists only in [osu!standard](/wiki/Game_Modes/osu!) and [osu!catch](/wiki/Game_Modes/osu!catch). AR defines when [hit objects](/wiki/Hit_Objects) start to fade in relatively to when they should be hit or collected. AR values range from 0 to 10. Higher approach rates mean that hit objects will be shown for a shorter period of time, thus giving less reaction time to plan ahead. Inversely, lower approach rates mean that hit objects will be shown on the screen for a longer time, allowing more time to react, but can result in an excessive amount of hit objects appearing on the screen at once. The duration of a hit object that stays visible on the screen (without mods) ranges from 1800ms at AR0 to 450ms at AR10.
+
+See the infographic and formulas below for an overview:
+
+```
+                                       X = hit/collect
+             p r e e m p t             ↓
+├───────────────────────┬──────────────┤
+0%       fade_in          100% opacity
+```
+
+The circle starts fading in at `X - preempt` with:
+
+- AR < 5: `preempt = 1200ms + 600ms * (5 - AR) / 5`
+- AR = 5: `preempt = 1200ms`
+- AR > 5: `preempt = 1200ms - 750ms * (AR - 5) / 5`
+
+The amount of time it takes for the hit object to completely fade in is also reliant on the approach rate:
+
+- AR < 5: `fade_in = 800ms + 400ms * (5 - AR) / 5`
+- AR = 5: `fade_in = 800ms`
+- AR > 5: `fade_in = 800ms - 500ms * (AR - 5) / 5`
+
+## osu!taiko and osu!mania
+
+Approach rate is visible in osu!taiko and osu!mania, but does not do anything that affects gameplay. This is because scroll speed is used instead, which is based on BPM.
```

Comment:
```suggestion
Approach rate is visible when viewing osu!taiko or osu!mania beatmap information, but it does not affect gameplay. Each mode's scroll speed is determined by other factors.
```
the thing about BPM is (partially) wrong, so I think this is okay. could really use links about scroll speed for these modes if there's such a place to link to right now (if not, your TODO covers this already)